src/fileio/formatfactory.cxx

```diff
@@ -14,6 +14,10 @@
 #include <output.hxx>
 #include <string.h>
 
+#ifdef __cpp_lib_make_unique
```

Comment:
Should this code go into `utils.hxx` so that `make_unique` is always defined? It does mean putting `using std::make_unique` in a header file though... 
Is this a good time to introduce a namespace, so `bout::make_unique` is always defined, importing `std::make_unique` if needed? 
(happy for other suggestions for namespace)

Reply:
I was um-ing and ah-ing about that myself! It might be the cleanest thing to do though.

I'm happy with `bout` as the namespace, possibly even `bout::utils`